libs/shared/src/types/rate-limiting/config.types.ts

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
The files below are refactored for better modularity to prepare for the evaluate rate limit use-case supporting configuration typings that will be introduced soon.